### PR TITLE
HMIS-ACP-007 Missed Appointment Tracking Form Validations

### PR DESCRIPTION
**Pull request by SoniaRMK**
This PR implements the following validations:

- [x] MCH/MBCP sub-options
- [x] Other(Specify) fields in the form
- [x] Date returned and Follow-up date should not allow future dates

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml [hunk 1]
```diff
@@ -1,63 +1,112 @@
...
+			console.log($lastVisibleRow);
+			$lastVisibleRow.find('.add').show();
+		};
 
-				disableContainer('#90041');
-				disableContainer('#165398');
-				disableContainer('#164975');
-				disableContainer('#165399');
+		jq('.follow-up').change(function(){
+			<!-- $otherOption= jq(this).find('[id^="165100"]').find(":checkbox").val();
```
Comment (solemabrothers) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml]: @SoniaRMK please remove this commented code if it is not being used anywhere

**Comment by solemabrothers** [comment]
@SoniaRMK remove this commented code also here https://github.com/METS-Programme/openmrs-module-ugandaemr-poc/blob/ac3792d08cb0a82591ba44b4a6aa488a2448c31b/omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml#L91

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml (hunk 1 above)
Comment (SoniaRMK) [reply omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml]: I am going to remove it

Comment (SoniaRMK) [reply omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml]: @solemabrothers Done!

**Pull request merged**